doc/api/http.md

```diff
@@ -623,6 +624,31 @@ If `data` is specified, it is equivalent to calling
 If `callback` is specified, it will be called when the request stream
 is finished.
 
+### `request.destroy([error])`
+<!-- YAML
+added: REPLACEME
```

Comment:
I think this was added in a previous release, right?

Reply:
It's almost always been there, just didn't work well.

Reply:
@nodejs/documentation: What's proper here? This PR doesn't add these methods but does document tham. What should "added" be? `REPLACEME` or try to find the earliest version where it exists?

Reply:
This has existed for a long time through `OutgoingMessage.destroy` which `ClientRequest` inherits from. It's hard to say which exact version. It goes back at least 4 years.

How to handle? Not sure who usually handles the REPLACEME sections. @targos?

Reply:
I would say the correct thing is to find the earliest place it was publicly exposed. That's what we've done in the past, and that's what I'd expect to see as a reader of the docs.

Reply:
I don't think we have `added:` metadata for everything, do we? It seems like it could be left off if it is difficult to figure out and someone could fill it in later.

Reply:
I think this has existed for a very long time and it is difficult to figure out exactly when it was added.

@Trott: Maybe I remove `added:` and land this or do we need further consensus?

Reply:
It looks like there was a (successful, AFAICT) effort to provided `added:` info in all the docs. See https://github.com/nodejs/node/issues/6578. So given that, we probably should add it. Most (all?) of the `added:` info for very-early APIs was provided by @addaleax in https://github.com/nodejs/node/pull/7392. I'm not sure what she did exactly, but maybe she would know how to make a determination without too much difficulty?

Reply:
What I did was a lot of bisecting using `nvm install` and tons of `git blame`.

These methods were added in 8139bb24c1f72cdea41492bf3bf395a66e04104f, which was first released in v0.3.0.

Reply:
Added in https://github.com/nodejs/node/pull/32807/commits/7082213fefc98ae51a22effb5f25a486a5d364c2.